Official Code of Conduct for Matrix and IRC channels

SyntheticBird45 opened the pull request:
Help at resolving: #899 

## Why ?

Frustration have grown in the community regarding moderation and general health of the project. The community is vital to the Project, and it is important that an effort is made to promote an healthy place for developers and users to cooperate.

## The CoC

The following Code of Conduct is a modified version of the Covenant Contributor's CoC, that take into consideration the culture of the Monero space by redefining the desired rules often heard in discussions and by eliminating the social aspect described by the original document. This should, with common sense, help at resolving some situations in the future.

## Typos

This document may contains typo, I'm not English native. Reporting spelling and grammar mistakes, is greatly appreciated.

## Discussions

Any point of this document might be discussed and the PR will stay as a draft until consensus is found

Commit:
Adding a Code of Conduct for Matrix/IRC channels

Initial version, modified from Covenant Contributors. Took into account critics from sech1 and jeffro.

Commit:
some edits

Commit:
Several edits

- Edited bad behavior regarding sexual/pornography. Pornography sharing is inacceptable.
- Removed All I have to know I learned in kindergarten.
- Added "intend" and "do not intend" section, this should cover the expectations around political discussions and culture war.
- Added a community moderators section, regarding what is a moderator, how to become one and how to remove one.

Commit:
fixed typos

SyntheticBird45 committed:
another typo I forgot

SyntheticBird45 committed:
typo

nahuhh commented:
care to explain how this resolves #899?

SyntheticBird45 commented:
> care to explain how this resolves #899?

https://github.com/SyntheticBird45/meta/blob/fb56cbe5ab966bbc8da603481372966468bede20/CHANNELS_CODE_OF_CONDUCT.md?plain=1#L68

nahuhh commented:
That doesn't resolve moderation overreach. It's just a vague "mission statement". The rest of the document is written poorly "act stupidly". 

Are mods accountable for their actions? 
I've spoken in depth about solutions, and a CoC isn't one of them. This CoC is for users, not mods. Doesn't solve issue of mods power tripping in rooms they dont even participate in, or forcing their opinion on the entire community.

SyntheticBird45 commented:
> That doesn't resolve moderation overreach. [...] This CoC is for users, not mods.

The overall document permit users to know what behavior is considered a violation. If moderators are tripping, this document can be used by users to report power abuse by showing that their actions do not enter the several unacceptable behavior quoted. Thus the following sentence:

`[...] as well as setting the expectations of the community members regarding the general health of the channels.`

> Are mods accountable for their actions?

`Community moderators have the right and responsibility to discuss actions with other moderators 
when the community pleads for revisions.`

`Existing moderators can be removed, if pleaded by the community, 
at the moderators' discretion.`

You can understand by community pleading, any form of actions and/or communications with the intent of reporting an observed unfairness. It is pretty obvious that the removal of moderator is acted by the moderators themselves, as they are the one with permissions, and that generally going against the community opinion (without proper reasons) will quickly lead into people fleeing the channels.

> [...] "act stupidly"

You may be surprise to hear that after trying to discuss that with everyone on Matrix, this has been a recurrent expression. And I think it is fair to use as it shows that the community is expecting members to make efforts. 

> Doesn't solve issue of mods power tripping in rooms they dont even participate in, or forcing their opinion on the entire community.

If by *"in rooms they dont even participate in"*, you mean rooms outside of #monero-*:

`This Code of Conduct applies within The Monero Project's Matrix and Libera IRC channels`

If you happen to have beef with the moderators outside of monero space, it is not this CoC problem to resolve. However, if they do threaten to take actions against you based on behavior outside of the space, you can use the CoC doc to report a power abuse to the community. 

> It's just a vague "mission statement". The rest of the document is written poorly [...] I've spoken in depth about solutions, and a CoC isn't one of them. [...]

Agree to disagree

nahuhh commented:
> > That doesn't resolve moderation overreach. [...] This CoC is for users, not mods.
> 
> The overall document permit users to know what behavior is considered a violation. If moderators are tripping, this document can be used by users to report power abuse by showing that their actions do not enter the several unacceptable behavior quoted. Thus the following sentence:
> 

Mods  (specifically scott) are banning and muting users because they feel like it. Scott spent years feeling good about keeping rotten banned. I got rotten unbanned, and he's been on a revenge mission since.

Dan/bob was muted by scott for asking (in resolutions) for them to reverse ercicciones rage-ban. Erc was mad that i fixed the haveno room, and was omw to fix website and translation, so he banned me from them. Xmrscott told dan/bob to tell me that i should "ask jwinterm for screenshots" if i need access to finish my work.

> `[...] as well as setting the expectations of the community members regarding the general health of the channels.`
> 

its pretty simple. Common courtesy.
> > Are mods accountable for their actions?
> 
> `Existing moderators can be removed, if pleaded by the community, 
> at the moderators' discretion.
> ..x It is pretty obvious that the removal of moderator is acted by the moderators themselves, as they are the one with permissions,

wrong. only server admin can demote an admin and only admins can demote moderators. 
of course, you can self-demote. But no, moderators do not have the power to force out an admin. 

> 
> > [...] "act stupidly"
> 
> You may be surprise to hear that after trying to discuss that with everyone on Matrix, this has been a recurrent expression. And I think it is fair to use as it shows that the community is expecting members to make efforts. 


i dont care how many people said it. Stupid is "stupid" language.


> If by *"in rooms they dont even participate in"*, you mean rooms outside of #monero-*:
> 

No. I DO mean #monero- rooms.

- Example 1 -gui: Plowsof unbanned me in monero-gui, scott rebanned me without notice.
- Example 2 -gui: Selsta asked for scott to unban me in monero-gui, Scott refused, claiming (lies) that i has threatened community members lives.
- Example 3 -beef: rucknium (who was a mod of -beef) invited me to monero-beef. Scott decided randomly that it was in the communities best interest to ban me from -beef. Scott also banned me in #monero-beef on irc.
- Example 4 -beef: Rucknium spoke with scott and unbanned me in -beef. Scott rebanned me again, resulting in rucknium stepping down as mod and **leaving the room altogether**. 
- Example 5 -support: Scott and does not participate in #monero-support, yet he banned me from there too.
- Example 6 -comnunity: Luigi asked Scott to unban me in community. Scott refused, telling Luigi not to "act on emotion" (scott being the one whom was actually acting on emotion). Scott had been inactive for many, many months leading up to this and, afaik, still does not even attend meetings.
- example 7 -site: luigi reversed ericcione's bans on irc (the same ban that scott muted dan over) and erciccione stepped down as admin. now erc has sunk low enough to launch **dishonest** attacks against selsta and woodser on Twitter and reddit.

these are all #monero namespace. I am **only** banned in #monero- rooms. Not banned in feather, cake, no wallet left behind, haveno, xmrbazaar, wownero, particl, kernal, etc etc.

the Banhammer was actually **banned** by tobtoht from feather and kicked from cake by vik - both of which unbanned me when they found out i had been banned.

and ive _only_ been banned by scott (current admin), sgp (former), and erc (former). The latter 2 stepped down from their positions for the same reason - overstepping. 

> `This Code of Conduct applies within The Monero Project's Matrix and Libera IRC channels`

correct. not sure why

> If you happen to have beef with the moderators outside of monero space, it is not this CoC problem to resolve. However, if they do threaten to take actions against you based on behavior outside of the space, you can use the CoC doc to report a power abuse to the community. 
> 

not relevant. Im not banned anywhere else.

ghost commented:
> its pretty simple. Common courtesy.

> i dont care how many people said it. Stupid is "stupid" language.

You fucking jackass.

nahuhh commented:
> > its pretty simple. Common courtesy.
> 
> > i dont care how many people said it. Stupid is "stupid" language.
> 
> You fucking jackass.

Idek you

xmrscott commented:
There seems to unsurprisingly be some malicious outright lying by ofrnxmr in a weird attempt to argue against a changed CoC so I'll quickly correct a few of the many lies here:

> Mods (specifically scott) are banning and muting users because they feel like it. Scott spent years feeling good about keeping rotten banned. I got rotten unbanned, and he's been on a revenge mission since.

rotten was banned for several violations of the existing CoC / ToS and declining to agree upon being asked to adhere to them. He has since agreed to them and was subsequently unbanned which I am happy for. ofrnxmr had 0 influence in mods unbanning rotten, as much as ofrnxmr likes to make themselves the center of attention such as abusing the CCS as a donation address instead of milestones such as here: ["explicitly requesting donations for my existence, and not "payment" for milestones."](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/444)

> Dan/bob was muted by scott for asking (in resolutions) for them to reverse ercicciones rage-ban. 

dan was muted in -resolutions after being given a warning to stay on topic. The mute was removed when sufficient time had passed.

**Examples 1-2:**

ofrnxmr lied to moderators that he could not perform work with an IRC account and therefore should be granted a Matrix ban exception in -gui. He was shortly after found to have lied about unviability of using his IRC account and the Matrix mods unanimously removed the ban exception that was predicated on a lie to mods.

**Examples 3-4:**

ofrnxmr lies in the comment above that "Rucknium spoke with scott and unbanned me in -beef.". Rucknium did not actually speak before unbanning. Much like examples 1-2, there wasn't a good reason to grant exemptions. The ban was for monero-* consistency especially since someone who is known to blatantly lie shouldn't be in -beef as lies run counter to the intent of -beef.

![image](https://github.com/monero-project/meta/assets/165470978/5dce3693-8995-471d-814e-12c21c1693de)

**Example 5:**

xmrscott is in fact active across rooms including -support and has pointed people in -support to resources such as getmonero.org and monero.stackexchange.com where answers to questions can be indexed by search engines instead of hidden in a Matrix room

**Example 6:**

luigi asked without knowing context as ofrnxmr's modus operandi is usually to make requests of individual by telling half truths, or otherwise omitting or lying to get some desired reaction that wouldn't happen if full context was known as ofrnxmr also did with Datahoarder. Once context was known, the desire by luigi to have ofrnxmr unbanned in -beef was removed. Case in point the -beef ban remains in place.

In fact, as ofrnxmr was making an emotional outburst, luigi commented that "ofrnxmr you're not really helping your case"

**Example 7:**

"the same ban that scott muted dan over" See -resolutions comment prior to example comments.




> the Banhammer was actually banned by tobtoht from feather and kicked from cake by vik - both of which unbanned me when they found out i had been banned.

tobtoht also unbanned naphtha who was banned by several mods across several rooms for various LiberaChat violations including deaths threats. Therefore tobtoht does not make for a good appeal to authority.

> [sgp (former), and erc (former)] stepped down from their positions for the same reason - overstepping.

They did not in fact step down for overstepping. They had other reasons; ofrnxmr as usual likes to put lies in people's mouths.


I would generally recommend that people when discussing CoC not take ofrnxmr's comments too seriously based on past behavior like making a donation address request in the form of a CCS, generally lying, and notable members of the community like kayabaNerve calling ofrnxmr a bad actor by the existing CoC: ["By my own opinion, by the opinion of others, and I believe by ofrnxmr themselves, ofrnxmr is unrepentantly offensive. That does classify them as a bad actor per that definition."](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/444#note_24012). It should therefore be unsurprising that someone who is classified as a bad actor within the community would lie.

Undoubtedly ofrnxmr AKA nahuhh,someone who hates basic CoCs, or someone who has interests in enabling bad actors will try to make some other set of arguments based on further lies, but I believe I have commented enough here to demonstrate his fondness of lying to not need to expose every lie that ends up on this PR by such individuals.

nahuhh commented:
> There seems to unsurprisingly be some malicious outright lying by ofrnxmr in a weird attempt to argue against a changed CoC so I'll quickly correct a few of the many lies here:
> 
> > Mods (specifically scott) are banning and muting users because they feel like it. Scott spent years feeling good about keeping rotten banned. I got rotten unbanned, and he's been on a revenge mission since.
> 
> rotten was banned for several violations of the existing CoC / ToS and declining to agree upon being asked to adhere to them. He has since agreed to them and was subsequently unbanned which I am happy for. ofrnxmr had 0 influence in mods unbanning rotten

i literally wrote his response

> as much as ofrnxmr likes to make themselves the center of attention such as abusing the CCS as a donation address instead of milestones such as here: ["explicitly requesting donations for my existence, and not "payment" for milestones."](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/444)

that was a verbatim request from kaya

> > Dan/bob was muted by scott for asking (in resolutions) for them to reverse ercicciones rage-ban. 
> 
> dan was muted in -resolutions after being given a warning to stay on topic. The mute was removed when sufficient time had passed.


youre a liar.
> **Examples 1-2:**
> 
> ofrnxmr lied to moderators that he could not perform work with an IRC account and therefore should be granted a Matrix ban exception in -gui. He was shortly after found to have lied about unviability of using his IRC account and the Matrix mods unanimously removed the ban exception that was predicated on a lie to mods.


again. Youre a liar. I never made any such claim, and you have no power in gui on irc
@plowsof unbanned me on matrix side, and i dis not request it. Weeks after your re-ban, I had spoken with @selsta on irc, i did not request for selsta to unban. You _lied_ to selsta (in public) and claimed that i had recently threatened community members lives. (liar)

> **Examples 3-4:**
> 
> ofrnxmr lies in the comment above that "Rucknium spoke with scott and unbanned me in -beef.". Rucknium did not actually speak before unbanning. Much like examples 1-2, there wasn't a good reason to grant exemptions. The ban was for monero-* consistency especially since someone who is known to blatantly lie shouldn't be in -beef as lies run counter to the intent of -beef.

perhaps i was mistaken. I believe i recall rucknium saying that he had spoken with you. Either way, you banned me form beef because youre a bitch, and ruck left the room because youre a child

> **Example 5:**
> 
> xmrscott is in fact active across rooms including -support and has pointed people in -support to resources such as getmonero.org and monero.stackexchange.com where answers to questions can be indexed by search engines instead of hidden in a Matrix room

lmao. I spend damn near an entire year being incredibly active in that room and never say you once.


> **Example 6:**
> 
> luigi asked without knowing context as ofrnxmr's modus operandi is usually to make requests of individual by telling half truths, or otherwise omitting or lying to get some desired reaction that wouldn't happen if full context was known as ofrnxmr also did with Datahoarder. Once context was known, the desire by luigi to have ofrnxmr unbanned in -beef was removed. Case in point the -beef ban remains in place.

rucknium unbanned in beef, and i asked datahoarder and rucknium if it could be mirrored on irc.

> In fact, as ofrnxmr was making an emotional outburst, luigi commented that "ofrnxmr you're not really helping your case"

No, my outbursts arent emotional. You're a coward, thats just stating the facts. Youre the worst mod monero has EVER seen.

> **Example 7:**
> 
> "the same ban that scott muted dan over" See -resolutions comment prior to example comments.


> 
> 
> > the Banhammer was actually banned by tobtoht from feather and kicked from cake by vik - both of which unbanned me when they found out i had been banned.
> 
> tobtoht also unbanned naphtha who was banned by several mods across several rooms for various LiberaChat violations including deaths threats. Therefore tobtoht does not make for a good appeal to authority.

Speak for yourself. _everybody_ respects tobtoht more than they do you, in every facet. Better than you at _everything_, AND people respect him.

> > [sgp (former), and erc (former)] stepped down from their positions for the same reason - overstepping.
> 
> They did not in fact step down for overstepping. They had other reasons; ofrnxmr as usual likes to put lies in people's mouths.

wrong again. Erc wanted to play admin and refuse to contribute, so he was asked to step down. sgp stepped down because he was "unable to do the moderation that he wanted".l aka because his attempts to power trip were met with resistance by everyone in the community. 
Both of you did nothing to keep channels on topic. You just show up to ban people you dont like. Fkn weirdos. 

> I would generally recommend that people when discussing CoC not take ofrnxmr's comments too seriously based on past behavior like making a donation address request in the form of a CCS, generally lying, and notable members of the community like kayabaNerve calling ofrnxmr a bad actor by the existing CoC: ["By my own opinion, by the opinion of others, and I believe by ofrnxmr themselves, ofrnxmr is unrepentantly offensive. That does classify them as a bad actor per that definition."](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/444#note_24012). It should therefore be unsurprising that someone who is classified as a bad actor within the community would lie.

I would recommend you gfy. You and 20 bots agree. 
Im a bad actor? Because you clowns mislead kaya?

> Undoubtedly ofrnxmr AKA nahuhh,

Did anyone thing i was someone else?

> someone who hates basic CoCs, 

talk about putting words in mouths

> or someone who has interests in enabling bad actors 

🥲🥲 coming from the mod who lets socks spam the rooms and does absolutely nothing to moderate actual bad actors - and who mutes and bans contributors (not named ofrn)

> will try to make some other set of arguments based on further lies, 

only liar here is you, xmrscott. You have some serious issues. 
> but I believe I have commented enough here to demonstrate his fondness of lying to not need to expose every lie that ends up on this PR by such individuals.

enabling bad actors? why dont you tell the world who spammed ccs and monero- rooms with banofrn accounts

nahuhh commented:
@xmrscott is the most pathetically dishonest coward ive ever come across. 

lmao, wow.  

literally revised history. Every example i gave was factual and csn be verified by every person involved. 

xmrscott is a _terrible_ liar.  Like, he really just makes shit up and remembers whatever he wants to remember, tries to take things outbof context _and add lies to them_ "dan was off topic" lmfao what? Youre a fucked up guy 😆! lmao

@xmrscott lets have an election. I'm willing to bet the community votes your lying ass out

its not simply that you have no conflict management skills, youre the weakest person in any conversation. You abuse your power because without your banhammer, you're a weak person and nobody would ever follow your lead. Someone like _you_ can never lead anyone, not even yourself. You need to be coddled and pampered or you throw tantrums.
youre a weak little man and you always have been. -- and before you go off deflecting again, trying to accuse me of being emotional, im honestly laughing my ass at your wild lies and attempts to sound professional to cover them up. Youre a comedian

SyntheticBird45 commented:
@nahuhh @SamsungGalaxyPlayer 

As much as I like dipping into drama as a source of entertainment, this is beyond my understanding and I do not pretend to represent the perfect justice. It's only natural that you two argue about each other, knowing the past hostility. But I hope you will excuse me for ignoring your beef comments.

@nahuhh regarding your concern of a mod power abusing in the #monero-* space. As I said earlier, if this is merged, you'll be able to use this document as a resource for appeal, and mods will have to conform to this document, meaning your fear of scott making solo decision should end up bad in the eyes of other moderators (if this is unreasonable).

From what I understand of your comment, you're frustrated because it do not resolve your actual situation at all. I just want to say that I need to make assumptions regarding the current state, because otherwise we'll never do anything. Rome wasn't built in a day, and I consider this CoC as a good starting point for building a saner place. I know you don't like this CoC, but you'll always be able to make a PR with changes if you feel like it.

nahuhh commented:
> @nahuhh @SamsungGalaxyPlayer 

why tagging sgp?
> @nahuhh regarding your concern of a mod power abusing in the #monero-* space. As I said earlier, if this is merged, you'll be able to use this document as a resource for appeal,

We already have appeals, #monero-resolutions, which xmrscott mutes users and decides himself what he wants to do.

> and mods will have to conform to this document, meaning your fear of scott making solo decision should end up bad in the eyes of other moderators (if this is unreasonable).

I dont fear scott making solo decisions, its simply evidenced that he does, and does so against the will of community leaders such as rucknium, plowsof and selsta. 
Other mods are powerless to hold an admin accountable. 
In addition, a fee of the current mods are inactive, unelected, or clearly show bias (allowing banofrn sockpuppets to flood and derail community - a place if work).

> From what I understand of your comment, you're frustrated because it do not resolve your actual situation at all. 

This CoC falsely claims to "resolve 899".
899 is my "actual situation".
This does nothing to resolve 899 - mods who shouldnt be mods.

> I just want to say that I need to make assumptions regarding the current state, because otherwise we'll never do anything.

Another CoC for users was never necessary.

> Rome wasn't built in a day, and I consider this CoC as a good starting point for building a saner place.

Community ran just fine before scott started revenge banning.
Respect is earned, as is disrespect.
example: plowsof, selsta, tobtoht have earned their respect. Xmrscott forces you to respect him via moderation and is the direct cause of 3+ contributors leaving all #monero- rooms. 

> I know you don't like this CoC, 

Has nothing to do with "like". It claims to resolve an issue that it doesnt even begin to address. 
This CoC does nothing to stop mods from abusing their power, and says false things like "other mods decide". 

> but you'll always be able to make a PR with changes if you feel like it.

It's not currently merged and isnt needed.
we already follow a basic code of ethics (matrix/libera). This CoC is just a poorly written attempt to legitimize a group of bad actors attempts to censor, disrupt and move the target from "moderation issue" to "community members issue".

What is needed, is a set of rules and guidelines for mods to abide by, and a process to replace mods. 

Abuse of power is the issue, not laws for users.
I had _never_ been banned until scott and sgp wanted to silence me because they don't like being challenged. 

899 has nothing to do with users behavior, and everything to do with opportunistic admins and moderators abusing their positions to benefit themselves at the detriment to monero (contributors leaving).

SyntheticBird45 commented:
> This CoC is just a poorly written attempt to legitimize a group of bad actors attempts to censor, disrupt and move the target from "moderation issue" to "community members issue".
> This CoC does nothing to stop mods from abusing their power, and says false things like "other mods decide".
> Abuse of power is the issue, not laws for users.

This is absolutely not what it is. I'm not *supporting sgp* or *being against users*, knowing my own intents it is pretty hard to not consider these conclusions as lies or bad faith arguments. I've the impression you either select what you talk about or just straight up don't understand what you read. To speak frankly, it is already hard for me to decide on a personal level about you, as you seems to always be against whatever I say and sometimes say childish things (I did see your tweets). On the other end, I've friends that are friend with you and are always telling me you're someone right. This is disappointing. And this is my personal opinion, not some *sgp-indoctrinated* one, as your message would probably make people think it is.

I did not expect to answer that way but I think it's necessary. I'm waiting for comments from core, and I'll ignore any beef one. As said earlier, agree to disagree.

Feel free to have the last word if you feel like it

nahuhh commented:
@SyntheticBird45  _why do you keep mentioning sgp??_

> just straight up don't understand what you read.

you don't even seem to know any context, which shows your making false claims about resolving 899 without knowing anything about it.

this is a CoC, written by you, that has no relation to past grievances of the community aside from whatever you tell yourself or, perhaps, what you have been told by people who make things up as they go

im not claiming you had bad intents, but you are trying to play the the "hero" and this CoC is even well written. I don't know why you woke up and decided that you should pretend to know what you're doing in regards to managing a workgroup or a group of workgroups. Wasnt curate previouslt _your_ workgroup? Why not try implementing a CoC there first? Why come over to monero to pretend to be some sort of hero, as if users were ever the problem.

PS. in case you were unaware, sgp and scott are diff ppl

ghost commented:
Ofrn criticizing this proposal over the quality of the writing is quite amusing when the overwhelming majority of his own could easily be mistaken for the drunken ramblings of a semi-literate child.

If anyone's concerns are actually just in the quality, concision, or coherence of the document itself then I'm happy to offer my editing abilities, something I have formal experience with in the past though not under this moniker.  Personally, however, I think its intention is perfectly clear as-is and suspect the critical parties are simply concern trolling.

SyntheticBird45 commented:
> If anyone's concerns are actually just in the quality, concision, or coherence of the document itself then I'm happy to offer my editing abilities, something I have formal experience with in the past though not under this moniker.

Any grammar correction would be greatly appreciated. You can make a PR on my repo: https://github.com/SyntheticBird45/meta/tree/master

Commit:
edits to CHANNELS_CODE_OF_CONDUCT.md

- fixed minor grammar errors
- changed members to participants across the board since nobody is carrying cards or paying dues
- avoided using terms such as rules, punishments, etc since they carry baggage
- kept an active tense throughout the document
- moved Scope section to directly follow the pledge for coherency, and collapsed the intended/not intended into scope for brevity

ghost commented:
I just opened the following PR: https://github.com/SyntheticBird45/meta/pull/1

SyntheticBird45 committed:
Merge pull request #1 from yasabi/master

edits to CHANNELS_CODE_OF_CONDUCT.md

SyntheticBird45 commented:
Merged @yasabi changes 👌

pm187 commented:
I am totally against any attempts to censor the discussion due to personal politics. This is how Bitcoin was captured, censored, handicapped, and continuously gatekept. Identity politics has absolutely nothing to do with software or free open source software development. The people trying to implement a CoC to stifle conversation and censor people they have political disagreements with are bad actors in my opinion and should not be trusted or perceived as credible, by anyone who has critical thinking skills.

SyntheticBird45 commented:
> I am totally against any attempts to censor the discussion due to personal politics. This is how Bitcoin was captured, censored, handicapped, and continuously gatekept. Identity politics has absolutely nothing to do with software or free open source software development. The people trying to implement a CoC to stifle conversation and censor people they have political disagreements with are bad actors in my opinion and should not be trusted or perceived as credible, by anyone who has critical thinking skills.

Thanks for confirming you didn't even read.
